scripts/install_dev_tools.sh

```diff
@@ -16,3 +16,8 @@ echo "poetry: $(poetry --version)"
 poetry install
 poetry lock --check
 npm ci
+
+# install cairo-rs-py
+git clone git@github.com:lambdaclass/cairo-rs-py.git
```

Comment:
Since this is an addition to dev tools, I guess the requirements for running starknet-devnet are also changed now? Potentially then `page/docs/intro.md` also needs to be changed.

Reply:
Yeah absolutely. I added to the requirements in the intro.md file